lib/reducers/sync-collections.js

```diff
@@ -0,0 +1,14 @@
+module.exports = (state, action) => {
+  const mergedCollections = {};
+  const { collections } = action.payload;
+  const collectionNames = Object.keys(collections);
+
+  collectionNames.forEach((collectionName) => {
+    mergedCollections[collectionName] = {
+      ...state[collectionName],
```

Comment:
I don't think we've added ES6 spreads as dependencies.

Let's switch these to `Object.assign`.